reposerver/repository/testdata/invalid-manifests/bad.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: ConfigMapList
+metadata: badData
+Here are invalid data
```

Comment:
Actually this still isn't testing the right thing since bad.yaml is invalid YAML. The test should have something that is:
1. valid yaml
2. looks like kubernetes yaml
3. fails to unmarshal.

```yaml
apiVersion: v1
kind: ConfigMapList
metadata:
  annotations:
    value-is-int: 1  # this should fail because annotations are supposed to be map[string]string
```

Reply:
Ignore requirement 1.

~1. valid yaml~